lib/rubocop/version.rb

```diff
@@ -44,15 +45,17 @@ def self.extension_versions(env)
         next unless (match = loaded_feature.match(/rubocop-(?<feature>.*)/))
 
         feature = match[:feature]
+
+        feature = 'markdown' if feature == 'md'
```

Comment:
I'd add a comment here explaining the need for this.

Reply:
Sure. Btw I'd rather provide a mapping like `CANONICAL_FEATURE_NAMES`, but couldn't think of a good name 🙈 
CANONICAL_PATH_NAMES perhaps?

Reply:
Or something like `EXTENSION_PATH_NAMES`?

Reply:
Perhaps we didn't need to include detection of rubocop-md in the first place, as it's not really an extension to rubocop (e.g. unlike the rest doesn't define new cops) 🤔